### PR TITLE
Add a new command-line argument to pass arguments to xacro

### PR DESCRIPTION
**Pull request by JafarAbdi**
Depends on https://github.com/openrr/urdf-rs/pull/61

Make it so it's possible to load xacro files that expects arguments.

### CODE REVIEW

File: src/utils.rs [hunk 1]
```diff
@@ -27,9 +27,9 @@ mod native {
 
     use crate::Result;
 
-    fn read_urdf(path: &str) -> Result<(urdf_rs::Robot, String)> {
+    fn read_urdf(path: &str, args: &[(String, String)]) -> Result<(urdf_rs::Robot, String)> {
```
Comment (OTL) [inline src/utils.rs:30]: urdf does not accept args. args is only for xacro.
So how about rename to xacro_args?

```suggestion
    fn read_urdf(path: &str, xacro_args: &[(String, String)]) -> Result<(urdf_rs::Robot, String)> {
```

**Review by OTL — COMMENTED** [review]
Thanks!

File: src/utils.rs [hunk 2]
```diff
@@ -27,9 +27,9 @@ mod native {
 
     use crate::Result;
 
-    fn read_urdf(path: &str) -> Result<(urdf_rs::Robot, String)> {
+    fn read_urdf(path: &str, xacro_args: &[(String, String)]) -> Result<(urdf_rs::Robot, String)> {
         let urdf_text = if Path::new(path).extension().and_then(OsStr::to_str) == Some("xacro") {
-            urdf_rs::utils::convert_xacro_to_urdf(path)?
+            urdf_rs::utils::convert_xacro_to_urdf_with_args(path, args)?
```
Comment (JafarAbdi) [inline src/utils.rs:32]:
```suggestion
            urdf_rs::utils::convert_xacro_to_urdf_with_args(path, xacro_args)?
```

**Comment by JafarAbdi** [comment]
Oops, looks like the main branch has been broken for a while, I can't see https://openrr.github.io/urdf-viz/?urdf=https://raw.githubusercontent.com/openrr/urdf-viz/main/sample.urdf -- Last commit should fix it.

I still have no idea why macos is failing, though

**Comment by OTL** [comment]
Thanks! @taiki-e is this OK?

**Review by taiki-e — APPROVED** [review]

**Pull request merged**